frontend/src/app/pages/MainPage/pages/SourcePage/SourceDetailPage/index.tsx

```diff
@@ -527,7 +526,7 @@ const Content = styled.div`
   }
 
   .detailForm {
-    max-width: ${SPACE_TIMES(240)};
+    max-width: ${SPACE_TIMES(400)};
```

Comment:
Why change it to 400x?

Reply:
The current JDBC screen has the field labels truncated. See the last few fields below.

![image](https://user-images.githubusercontent.com/100477931/167085848-57dcd745-44c6-44a7-b0b6-7af6696bc2c6.png)

Reply:
> The current JDBC screen has the field labels truncated. See the last few fields below.

Roger that 👌